pages/teams.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react'
+import Router from 'next/router'
+import Section from '../components/section'
+import Table from '../components/table'
+import { getTeams } from '../lib/teams-api'
+
+export default class TeamList extends Component {
+  static async getInitialProps ({ query }) {}
+
+  constructor (props) {
+    super(props)
+    this.state = {
+      loading: true,
+      error: undefined
+    }
+  }
+
+  async componentDidMount () {
+    try {
+      const teams = await getTeams()
+      this.setState({
+        teams,
+        loading: false
+      })
+    } catch (e) {
+      console.error(e)
+      this.setState({
+        error: e,
+        teamsk: [],
+        loading: false
+      })
+    }
+  }
+
+  renderTeams () {
+    const { teams } = this.state
+    if (!teams) return null
+
+    if (teams.length === 0) {
+      return <p className='measure-copy'>No teams created</p>
+    }
+
+    return (
+      <Table
+        rows={teams}
+        columns={[
+          { key: 'id' },
+          { key: 'name' },
+          { key: 'hashtag' }
+        ]}
+        onRowClick={(row, index) => {
+          Router.push(`/team?id=${row.id}`, `/team/${row.id}`)
```

Comment:
@kamicut maybe an individual team page should be available at `/teams/:id` rather than `/team/:id`.

```suggestion
          Router.push(`/team?id=${row.id}`, `/teams/${row.id}`)
```

Reply:
Yeah I thought about it, but we would have to rename the `pages/team.js` component to `pages/teams` or we could also change it in the router such that `/teams/id` renders `team.js`. What do you think?

Reply:
Let's do this as a followup if we change it since it involves a few out of scope changes for this pr.